setup.py

```diff
@@ -2,8 +2,8 @@
 import versioneer
 
 install_requires = ['menpo>=0.7,<0.8',
-                    'cyrasterize>=0.2.4,<0.3',
-                    'mayavi>=4.4.4']
+                    'cyrasterize>=0.3,<0.4',
+                    'mayavi>=4.5.0']
```

Comment:
Well I'm a little hazy on that tbh, I think I've seen conda not elect to bump an optional dep if the dep is already satisfied for the user (e.g. a current menpo3d user with mayavi 4.4.4 wouldn't get mayavi 4.5.0 by default if they `conda install -c menpo menpo3d`). Quite possibly I'm mistaken though?